### PR TITLE
Add default value in Attention layer docs #50839

### PR DESCRIPTION
**Pull request by europeanplaice**
In https://www.tensorflow.org/api_docs/python/tf/keras/layers/Attention, default values of causal and dropout are not specified and a user have to look up in the source code.

I added default values to this document in this pull request.

### CODE REVIEW

**Review by qlzh727 — APPROVED** [review]
Thanks for the update.
